src/SwipeProfiles.js

```diff
@@ -1,194 +1,215 @@
-        var formData = new FormData();
-        formData.append("currentUserId", currentUserId);
-        formData.append("shownUserId", displayId);
-        formData.append("match", choice);
-        const myRequest = new Request('http://127.0.0.1:5000/swipe', {
-              method: 'POST',
-              body: formData,
-            });
-        fetch(myRequest)
-          .then(res => res.json())
-          .then(res => { 
-            if(res.response === "Success") {
-              var potentialList = this.state.potentialFriends;
-              var newPotentialUserId = potentialList.pop();
-              this.setState({
-                potentialFriends: potentialList,
-                displayedUserId: newPotentialUserId,
-                error: '',
-              });
-              this.displayProfile();
-            } 
-            else{             
-              this.setState({
-                error: res.response
-              });
-          }
-          })
-          .catch((error) => {
-              this.setState({
-                error: "Error connecting to backend"
-              });
-            }
-          );
+  render() {
+    const potentialFriends = this.state.potentialFriends;
+    const displayedUserId = this.state.displayedUserId;
+    const error = this.state.error;
+    if (
+      displayedUserId === "" &&
+      potentialFriends.length === 0 &&
+      error === ""
+    ) {
+      this.getPotentialFriendList();
     }
 
-    render(){
-      const potentialFriends = this.state.potentialFriends;
-      const displayedUserId = this.state.displayedUserId;
-      const error = this.state.error;
-      if(displayedUserId === "" && potentialFriends.length === 0 && error ==="") {
-        this.getPotentialFriendList();
-      }
-      
-       return(
-          <div className = "SwipeProfile">
-            <header>Potential Friends!</header>  
-            <Link to={{pathname: '/matches', state: {id: this.state.id}}}>See matches</Link>
+    return (
+      /*navigation bar and other necessary information about the match*/
+      <div className="SwipeProfile">
+        <ul>
+          <li>
+            <a href="Matches.js">Matches</a>
+          </li>
+          <li>
+            <a href="profile.js">My Profile</a>
+          </li>
+          <li>
+            <a href="Settings">Settings</a>
+          </li>
+        </ul>
+        <br></br>
+        <header class="pageTitle">Potential Friends!</header>
+        <br></br>
+        <img src="ppFriendsLogo.png"></img>
+        <br></br>
+        <Link to={{ pathname: "/matches", state: { id: this.state.id } }}>
+          See matches
+        </Link>
+        <br></br>
+        <br></br>
+        {error ? (
+          <text>{error}</text>
+        ) : (
+          <div>
+            <p>Name: </p>
+            <text>{this.state.firstName}</text>
+            <p>Age: </p>
+            <text>{this.state.age}</text>
```

Comment:
include this section in the profile intro section? it looks a little odd when its separated by the turquoise section

Reply:
It’s done intentionally - a lot of apps these days separate name and age with the rest of the profile details (ie. tinder). It helps some info stand out more than others, in this case, name and age is what I’d want the user to first see when they view their potential matches